app/models.py

```diff
@@ -1417,10 +1436,13 @@ class BriefResponse(db.Model):
 
     created_at = db.Column(db.DateTime, index=True, nullable=False, default=datetime.utcnow)
     submitted_at = db.Column(db.DateTime, nullable=True)
+    awarded_at = db.Column(db.DateTime, nullable=True)
 
     brief = db.relationship('Brief', lazy='joined')
     supplier = db.relationship('Supplier', lazy='joined')
 
+    award_details = db.Column(JSON, nullable=True)
```

Comment:
I'd probably stick this above the relationships, group  everything that's an actual column in the db